doc/examples/segmentation/plot_regionprops.py

```diff
@@ -97,32 +97,44 @@
 mask = morphology.remove_small_holes(mask, 50)
 labels = measure.label(mask)
 
+property_names = ["label", "area", "eccentricity", "perimeter", "intensity_mean"]
+
+props = pd.DataFrame(measure.regionprops_table(labels, img, properties=property_names))
+
+# Filter out big regions
+props = props[props["area"] < 5000]
+
 fig = px.imshow(img, binary_string=True)
-fig.update_traces(hoverinfo='skip')  # hover is only for label info
+fig.update_traces(hoverinfo="skip")  # hover is only for label info
 
-props = measure.regionprops(labels, img)
-properties = ['area', 'eccentricity', 'perimeter', 'intensity_mean']
+# Prepare data for scatter plot
+scatter_data = []
+for label_i in props["label"]:
+    contour = measure.find_contours(labels == label_i, 0.5)[0]
+    y, x = contour.T
+    hoverinfo = "<br>".join(
+        [
+            f'<b>{prop_name}: {props.loc[props["label"] == label_i, prop_name].values[0]:.2f}</b>'
+            for prop_name in property_names
+        ]
+    )
+    scatter_data.append({"x": x, "y": y, "label": label_i, "hoverinfo": hoverinfo})
 
 # For each label, add a filled scatter trace for its contour,
 # and display the properties of the label in the hover of this trace.
-for index in range(1, labels.max()):
-    label_i = props[index].label
-    contour = measure.find_contours(labels == label_i, 0.5)[0]
-    y, x = contour.T
-    hoverinfo = ''
-    for prop_name in properties:
-        hoverinfo += f'<b>{prop_name}: {getattr(props[index], prop_name):.2f}</b><br>'
+for data_i in scatter_data:
     fig.add_trace(
         go.Scatter(
-            x=x,
-            y=y,
-            name=label_i,
-            mode='lines',
-            fill='toself',
+            x=data_i["x"],
+            y=data_i["y"],
+            name=str(data_i["label"]),
+            mode="lines",
+            fill="toself",
             showlegend=False,
-            hovertemplate=hoverinfo,
-            hoveron='points+fills',
+            hovertemplate=data_i["hoverinfo"],
+            hoveron="points+fills",
         )
     )
 
-plotly.io.show(fig)
+fig.update_layout(autosize=True)
+fig.show()
```

Comment:
Checking the [rendered HTML if this example](https://output.circle-artifacts.com/output/job/20b8a487-b578-49af-a46d-27e27c45dafc/artifacts/0/doc/build/html/auto_examples/segmentation/plot_regionprops.html), it seems like the plot isn't shown.

I think should do it?
```suggestion
plt.show()
```

Reply:
This is surprising! the plot renders on the jupyter notebook just fine wih fig.show(). Plotly docs suggest changing the renderer to one of 'notebook' or 'browser' should work. I just reverted back to the previous version using plotly.io which worked.

Reply:
Maybe a quirk of how sphinx_gallery interacts with matplotlib?

That said, I'd actually prefer to use matplotlib over plotly in this case since there's no big benefit to having an interactive plot here. Plotly significantly increases the memory footprint of the rendered HTML and sometimes leads to degraded performance when just viewing the example.

But since this was using plotly before that's not blocking approval for me. :) Though, I wouldn't mind if you reverted to using matplotlib again. :sweat_smile: 

> Plotly docs suggest [...]

Not sure how matplotlib relates to this?

Reply:
Hmm, the `hoverinfo` is nice, though!

Reply:
The example's motivation says "It is also possible to explore interactively the properties of labelled objects by visualizing them in the hover information of the labels. This example uses plotly in order to display properties when hovering over the objects."!!